.github/workflows/main.yml

```diff
@@ -196,39 +200,187 @@ jobs:
 
-        #TODO: test low pu combine (and combined fit)
-          #- name: lowpu w combine
-          #  run: scripts/ci/run_with_singularity.sh scripts/ci/setup_and_run_python.sh scripts/histmakers/setupLowPileupCombine.py -i lowPU_mu_maxFiles2.hdf5 --verbose 4
+      - name: lowpu w mu plot mt
+        run: scripts/ci/run_with_singularity.sh scripts/ci/setup_and_run_python.sh scripts/plotting/makeDataMCStackPlot.py --rebin 2 --baseName transverseMass --hists mt -a mu -p $WEB_DIR -f $PLOT_DIR/lowPU $HIST_FILE_MU
 
-      - name: lowpu z analysis
-        run: scripts/ci/run_with_singularity.sh scripts/ci/setup_and_run_python.sh scripts/histmakers/mz_lowPU.py -o $WREMNANTS_OUTDIR -j $NTHREADS --maxFiles $MAX_FILES --verbose 4 --forceDefaultName
+      - name: lowpu w mu combine mt setup
+        run: scripts/ci/run_with_singularity.sh scripts/ci/setup_and_run_python.sh scripts/combine/setupCombineWMass.py -i $HIST_FILE_MU --verbose 4 --baseName transverseMass --fitvar mt --noEfficiencyUnc -o $WREMNANTS_OUTDIR
 
-      - name: lowpu z combine setup
-        run: scripts/ci/run_with_singularity.sh scripts/ci/setup_and_run_python.sh scripts/combine/setupLowPU_Z.py -i $HIST_FILE --verbose 4 -o $WREMNANTS_OUTDIR
+      - name: lowpu w mu plot response matrix
+        run: scripts/ci/run_with_singularity.sh scripts/ci/setup_and_run_python.sh scripts/plotting/response_matrix.py --axes ptll-ptVGen --procFilters Wmunu -p mu -o $WEB_DIR -f $PLOT_DIR/lowPU $HIST_FILE_MU
 
-          # FIXME: Remove from CI for now, to be added back in Jan's next PR
-          #- name: lowpu z combine fit
+      - name: lowpu w mu combine unfolding setup
+        run: scripts/ci/run_with_singularity.sh scripts/ci/setup_and_run_python.sh scripts/combine/setupCombineWMass.py -i $HIST_FILE_MU --verbose 4 --unfolding --fitvar ptll --noEfficiencyUnc -o $WREMNANTS_OUTDIR
 
-          #  run: cmssw-cc7 --command-to-run scripts/ci/run_combine.sh /home/c/cmsmwbot/combinetf/CMSSW_10_6_30/src/ CombineStudies/lowPU_differential/ lowPU_Zmumu_RawPFMET_differential.txt
+      - name: lowpu w e analysis
+        run: scripts/ci/run_with_singularity.sh scripts/ci/setup_and_run_python.sh scripts/histmakers/mw_lowPU.py -o $WREMNANTS_OUTDIR -j $NTHREADS --verbose 4 --forceDefaultName --unfolding --noRecoil --flavor e
+
+      - name: lowpu w e plot ptll
+        run: scripts/ci/run_with_singularity.sh scripts/ci/setup_and_run_python.sh scripts/plotting/makeDataMCStackPlot.py --hists ptll -p $WEB_DIR -f $PLOT_DIR/lowPU -a e $HIST_FILE_E variation --varName nominal_uncorr --varLabel MiNNLO --colors red
+
+      - name: lowpu w e plot pt
+        run: scripts/ci/run_with_singularity.sh scripts/ci/setup_and_run_python.sh scripts/plotting/makeDataMCStackPlot.py --baseName lep_pt --hists pt -a e -p $WEB_DIR -f $PLOT_DIR/lowPU $HIST_FILE_E
+
+      - name: lowpu w e plot eta
```

Comment:
Why can't you plot some of these observables in the same command?

Reply:
I think we would need to have them in a single histogram as different axes, no?

Reply:
hmm yeah you're right... it wouldn't be very hard to make baseName have nargs=+, but not sure it's worth it.

Reply:
Another option is just to have a shell script to make all the low pu plots, can also be useful for running outside the CI